specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchservice.json

```diff
@@ -6943,6 +6943,92 @@
       },
       "description": "Base type for skills."
     },
+    "CustomEntity": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The top-level entity descriptor. Matches in the skill output will be grouped by this name, and it should represent the \"normalized\" form of the text being found."
+        },
+        "description": {
+          "type": "string",
+          "description": "This field can be used as a passthrough for custom metadata about the matched text(s). The value of this field will appear with every match of its entity in the skill output."
+        },
+        "type": {
+          "type": "string",
+          "description": "This field can be used as a passthrough for custom metadata about the matched text(s). The value of this field will appear with every match of its entity in the skill output."
+        },
+        "subtype": {
+          "type": "string",
+          "description": "This field can be used as a passthrough for custom metadata about the matched text(s). The value of this field will appear with every match of its entity in the skill output."
+        },
+        "id": {
+          "type": "string",
+          "description": "This field can be used as a passthrough for custom metadata about the matched text(s). The value of this field will appear with every match of its entity in the skill output."
+        },
+        "caseSensitive": {
+          "type": "boolean",
+          "description": "Defaults to false. Boolean value denoting whether comparisons with the entity name should be sensitive to character casing. Sample case insensitive matches of \"Microsoft\" could be: microsoft, microSoft, MICROSOFT."
```

Comment:
You should specify the default value in swagger as well.

Reply:
There are optional parameters. If customers don't provide any value, our backend service will get a null, which is an equivalence of "false".